codespell_lib/data/dictionary.txt

```diff
@@ -19619,7 +19632,7 @@ peirod->period
 peirodical->periodical
 peirodicals->periodicals
 peirods->periods
-Peloponnes->Peloponnesus
+peloponnes->Peloponnese, Peloponnesus,
```

Comment:
Sorry for changing my mind, can you keep the case of the typo as the system lowercases it.

Reply:
done. no problem :smile: